### ISSUE TITLE
Keyboard navigation doesnt work with Select

### ISSUE THREAD

**Issue by a2441918** [palantir/blueprint#4068]
#### Environment

- __Package version(s)__:      
    "@blueprintjs/core": "3.25.0",
    "@blueprintjs/icons": "^3.15.0",
    "@blueprintjs/select": "3.12.1",
- __Browser and OS versions__: 
    "Chrome"

#### Question

Hi, I implemented Select but for some reason keyboard navigation isnt working. I replicated the same one as from the example. What am I missing? Please advice.

[https://stackblitz.com/edit/react-ts-w12ule?file=countries.tsx](https://stackblitz.com/edit/react-ts-w12ule?file=countries.tsx)

**Comment by cybae0804** [comment]
Same issue here

**Comment by a2441918** [comment]
@cybae0804
